src/Microsoft.AspNetCore.Authentication.JwtBearer/JwtBearerHandler.cs

```diff
@@ -185,11 +185,20 @@ internal class JwtBearerHandler : AuthenticationHandler<JwtBearerOptions>
 
         protected override async Task<bool> HandleUnauthorizedAsync(ChallengeContext context)
         {
-            var authResult = await HandleAuthenticateOnceAsync();
+            Exception failure = null;
```

Comment:
I think this is reasonable behavior to break, @Tratcher thoughts?  I don't think its reasonable for people to depend on random uncaught exceptions